src/styles/globals.css

```diff
@@ -0,0 +1,82 @@
+@import "tailwindcss";
+
+/* Base Styles */
+html {
+  scroll-behavior: smooth;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  /*@apply text-main bg-background font-inter font-normal;*/
+  overflow-x: hidden;
+  overflow-y: auto;
+  height: 100%;
+  margin: 0;
+}
+
+/* Container */
+.container {
+  @apply mx-auto px-4 w-full;
+  max-width: 375px;
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 768px;
+  }
+}
+
+@media (min-width: 1680px) {
+  .container {
+    max-width: 1680px;
+  }
+}
+
+.home-container {
+  max-width: 1680px;
+  margin: 0 auto;
+  padding-bottom: 40px;
+}
+
+.section-container {
+  padding-top: 36px;
+  padding-bottom: 36px;
+}
```

Comment:
you can also define these classnames in the same config.
Or, probably tailwind doesn't support that. 
But they have utilities that is basically what i'm talking about: https://tailwindcss.com/docs/adding-custom-styles
https://tailwindcss.com/docs/functions-and-directives